src/components/OAuthConnectButton.vue

```diff
@@ -43,6 +43,14 @@ export default {
 
 	computed: {
 		adminConfigNotOkMessage() {
+			if (getCurrentUser().isAdmin) {
+				const linkText = t('integration_openproject', 'Administration Settings > OpenProject')
+				const url = generateUrl('/settings/admin/openproject')
+				const htmlLink = `<a class="link" href="${url}" target="_blank" title="${linkText}">${linkText}</a>`
+				const hintText = t('integration_openproject', 'Some OpenProject integration application settings are not working. '
+				+ 'Click here {htmlLink} to set them up properly.', { htmlLink }, null, { escape: false, sanitize: false })
```

Comment:
Maybe something like that:

```suggestion
				const hintText = t('integration_openproject', 'Some OpenProject integration application settings are not working. '
				+ 'Configure the OpenProject integration in: {htmlLink}.', { htmlLink }, null, { escape: false, sanitize: false })
```